---
title: 例外 Message5 の完了 |Microsoft Docs
ms.custom: ''
ms.date: 06/08/2017
ms.prod: biztalk-server
ms.reviewer: ''
ms.suite: ''
ms.tgt_pltfrm: ''
ms.topic: article
helpviewer_keywords:
- exceptions, completing the flow
ms.assetid: 4aad3ff6-66d7-41e4-9079-a32149ad0a30
caps.latest.revision: 5
author: MandiOhlinger
ms.author: mandia
manager: anneta
ms.openlocfilehash: 05526dcfcea78b0a034b2eac4359116e61015fa5
ms.sourcegitcommit: 381e83d43796a345488d54b3f7413e11d56ad7be
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 05/07/2019
ms.locfileid: "65356812"
---
# <a name="completing-the-exception-message"></a><span data-ttu-id="c36d6-102">例外完了メッセージ</span><span class="sxs-lookup"><span data-stu-id="c36d6-102">Completing the Exception Message</span></span>
<span data-ttu-id="c36d6-103">例外のフローを完了するには、エラー メッセージ _ 3 を監視対象フォルダーにリダイレクトする送信図形を作成します。</span><span class="sxs-lookup"><span data-stu-id="c36d6-103">To complete the flow of the exception, create a Send shape to redirect the fault Message_3 to a monitored folder.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="c36d6-104">参照</span><span class="sxs-lookup"><span data-stu-id="c36d6-104">See Also</span></span>  
 <span data-ttu-id="c36d6-105">[例外のキャッチ ブロックを追加する方法](../core/how-to-add-a-catch-exception-block1.md) </span><span class="sxs-lookup"><span data-stu-id="c36d6-105">[How to Add a Catch Exception Block](../core/how-to-add-a-catch-exception-block1.md) </span></span>  
 [<span data-ttu-id="c36d6-106">BizTalk Server 例外処理の使用</span><span class="sxs-lookup"><span data-stu-id="c36d6-106">Using BizTalk Server Exception Handling</span></span>](../core/using-biztalk-server-exception-handling3.md)